src/SSHLibrary/abstractclient.py

```diff
@@ -42,6 +44,7 @@ def __init__(self, host, alias, port, timeout, newline, prompt, term_type,
             timeout=TimeEntry(timeout),
             newline=NewlineEntry(newline),
             prompt=StringEntry(prompt),
+            prompt_is_regexp=StringEntry(prompt_is_regexp),
```

Comment:
Would it be more appropriate to introduce new `BooleanEntry`, which would do the `is_truthy` magic, and stuff later here in `_read_login_output` and `login_with_public_key` would not need to repeat `is_truthy` every time?

Reply:
Great suggestion!